RELEASE.md

```diff
@@ -16,3 +16,5 @@
 4. Update the Helm Chart:
    - Repository https://github.com/grafana/helm-charts/tree/main/charts/rollout-operator
    - [Example PR](https://github.com/grafana/helm-charts/pull/3177/files)
+
+5. Update [images.libsonnet] (operations/rollout-operator/images.libsonnet)
```

Comment:
Should this go between the current steps 1 and 2 so that it's included in the tagged version?